Firestore/Protos/CMakeLists.txt

```diff
@@ -306,3 +326,95 @@ add_custom_target(
     generate_cpp_protos
     generate_objc_protos
 )
+
+if(APPLE)
+  set(
+    protobuf_objc_source_dir
+    ${FIREBASE_EXTERNAL_SOURCE_DIR}/grpc/third_party/protobuf/objectivec
+  )
+  objc_framework(
+    Protobuf
+    HEADERS
+      ${protobuf_objc_source_dir}/GPBArray.h
```

Comment:
This runs the risk of becoming stale wrt upstream. But IIUC this is temporary, so it seems reasonable.